src/main/java/bio/terra/common/DaoUtils.java

```diff
@@ -32,16 +33,24 @@ public static String orderByClause(EnumerateSortByParam sort, SqlSortDirection d
             .toString();
     }
 
-    public static void addFilterClause(String filter, MapSqlParameterSource params, List<String> clauses) {
+    public static void addFilterClause(String filter, MapSqlParameterSource params, List<String> clauses,
+                                              String datasetIdField) {
         if (!StringUtils.isEmpty(filter)) {
-            params.addValue("filter", DaoUtils.escapeFilter(filter));
-            clauses.add(" (name ILIKE :filter OR description ILIKE :filter) ");
+            GoogleRegion regionFilter = GoogleRegion.fromValue(filter);
+            if (regionFilter != null) {
```

Comment:
I thought it would be clearer to have one `filter` since it already combines the name and description, but I can see how separating that into a region filter makes the behavior more intuitive. I'll look into that!